src/core/include/openvino/core/visibility.hpp

```diff
@@ -77,31 +77,3 @@
 #else
 #    define OV_NO_DANGLING
 #endif
-
-#if !(defined(_MSC_VER) && __cplusplus == 199711L)
-#    if __cplusplus >= 201103L
-#        define OPENVINO_CPP_VER_11
-#        if __cplusplus >= 201402L
-#            define OPENVINO_CPP_VER_14
-#            if __cplusplus >= 201703L
-#                define OPENVINO_CPP_VER_17
-#                if __cplusplus >= 202002L
-#                    define OPENVINO_CPP_VER_20
-#                endif
-#            endif
-#        endif
-#    endif
-#elif defined(_MSC_VER) && __cplusplus == 199711L
-#    if _MSVC_LANG >= 201103L
-#        define OPENVINO_CPP_VER_11
-#        if _MSVC_LANG >= 201402L
-#            define OPENVINO_CPP_VER_14
-#            if _MSVC_LANG >= 201703L
-#                define OPENVINO_CPP_VER_17
-#                if _MSVC_LANG >= 202002L
-#                    define OPENVINO_CPP_VER_20
-#                endif
-#            endif
-#        endif
-#    endif
-#endif
```

Comment:
let's keep it here, it's used in other places

Reply:
Done